natlas-server/app/models/__init__.py

```diff
@@ -2,24 +2,24 @@
 from app.models.agent_config import AgentConfig
 from app.models.agent_script import AgentScript
 from app.models.config_item import ConfigItem
-from app.models.email_token import EmailToken
 from app.models.natlas_services import NatlasServices
 from app.models.rescan_task import RescanTask
 from app.models.scope_item import ScopeItem
 from app.models.tag import Tag
 from app.models.user import User
+from app.models.user_invitation import UserInvitation
 from app.models.scope_log import ScopeLog
```

Comment:
Maybe my Python fu is weak, but why do we import these if __all__ is just referencing strings, not the actual class itself.

Reply:
Yeah I'm not entirely sure why this decision was made, but `__all__`  does use strings instead of the classes. `__all__` is basically just a way to say "this is the list of module names that should be imported if you do `from app.models import *`". I try to avoid importing * at all because it can have unwanted side-effects of namespace collisions, but supplying `_all__` explicitly does mean that if we wanted to add additional things to this `__init__.py` script, we could do it without worrying about any other code accidentally having namespace collisions.

These imports are really just here to make imports elsewhere shorter, anyways. `from app.models import User` instead of always having to do `from app.models.user import User`